### ISSUE TITLE
Search results unreadable

### ISSUE THREAD

**Issue by MidwestPat** [PhilippC/keepass2android#1435]
Love the app! Normally works really well.
For the past few days, when I have the app and my database open, and type in a search - the search results appear as a white list with no readable text. I cannot see the labels to choose the entry I am searching for. I use this feature constantly! Please advise.

![20201003_164114 (2)](https://user-images.githubusercontent.com/72320473/95001462-335f8d00-0598-11eb-84d1-263e85f3d71e.jpg)

**Comment by PhilippC** [comment]
this doesn't look good  :-)
I suspect there is some entry in your database which is causing this behavior. Can you try to create a new (test/demo) database and see if it happens there as well?

**Comment by BadGolfer2102** [comment]
This happened to me as well.  It seems the search results with a font of white on white.  The search is still filtering records because the search results box becomes smaller to indicate that it is filtering the results.  I can also 'select' an entry in the list.  The workaround so far is a reboot of the phone.

**Comment by MidwestPat** [comment]
Yes, a reboot ultimately fixed mine as well.

**Comment by BadGolfer2102** [comment]
FYI. The reboot doesn't permanently fix the issue for me.

**Comment by PhilippC** [comment]
can you tell when this starts to happen again then?

**Comment by billhuber** [comment]
I am having the same problem. When you are entering your search, the suggestions are white on white. This is true if you are using the If you press the Go key, the search results are visible. A similar problem occurs with the settings menus if you try the dark theme. The settings menu will be white on white.

**Comment by Pebberman** [comment]
Hello, the same for me, both in the general options and in the search results. 
Android 11, OneUI 3.0

**Comment by jryberg** [comment]
I have this issue as well. Samsung S21

**Comment by ChristophV-art** [comment]
Same here, Samsung S21 Ultra (Android11, One UI 3.1), changing design or reboot didn't fix anything for me. Seachbox remains white background with white font. Please fix, cheers

**Comment by JetCheOff** [comment]
Having the same issue, white text on white background. Can't say for sure but it seem to have started with Android 11.

**Comment by scottlamers** [comment]
I am having the same issue with my Samsung S21...  White on white and essentially unreadable. Any Help is appreciated. Reboot seems to fix it the first time you open, but then subsequent searches are unreadable again.

**Comment by MidwestPat** [comment]
Yes - the problem now seems to get fixed only the first time you open it
after a reboot. This is new (to me)  not a symptom originally reported. And
that only started with the most recent OS upgrade.

Same here.

**Comment by scottlamers** [comment]
I retract what I said. Rebooting does not make it work even once.. I must have done something else.

**Comment by AlphaDE** [comment]
Same Issue here Samsung S7+, Android 11 with all text in menus or search results, white on white regardless of white or dark mode.

**Comment by LuLei60** [comment]
Hi Philipp, are there any activities to fix this issue? Same issue like AlphaDE on same hardware and OS combination

**Comment by PhilippC** [comment]
I was hoping this will be fixed by Samsung as they did roll out fixes to some devices. Unfortunately I don't have a device for testing.

**Comment by scottlamers** [comment]
In the past week or so I have noticed that this is now working as expected..

**Comment by LuLei60** [comment]
> I was hoping this will be fixed by Samsung as they did roll out fixes to some devices. Unfortunately I don't have a device for testing.

Dear Philipp, Scott 
I'm sorry, but it is not working on my S7+. I have check my OS. Newest Samsung patches are installed

![SmartSelect_20210703-173119_Software update](https://user-images.githubusercontent.com/86771047/124359665-42a66080-dc26-11eb-8d6b-bdc700e754db.jpg)

**Comment by scottlamers** [comment]
It is working on my s21+ 5G.

**Comment by MidwestPat** [comment]
It comes and goes with various updates and rebooting the phone. I'm getting
by - still appreciate the app. I agree that it would be nice for Samsung to
fix it though.

**Comment by ChristophV-art** [comment]
Fixed for me since two or three updates ago by Samsung (not sure which one). S21 Ultra

**Comment by LuLei60** [comment]
Hi guys,
Today I got the update of KeePass2Android to 1.09a-r3. With this update the issue is fixed on my device
Thank you Philipp

**Comment by PhilippC** [comment]
sorry to say but this seems to be more a kind of random effect, I didn't actively change anything there...
